### PR TITLE
Add test cases for duplicate prefix

### PR DESCRIPTION
**Pull request by anir2212**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddTaskCommandParser.java [hunk 1]
```diff
@@ -24,6 +24,7 @@ public class AddTaskCommandParser implements Parser<AddTaskCommand> {
     public AddTaskCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_TASK_NAME, PREFIX_TASK_DESCRIPTION);
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_TASK_NAME, PREFIX_TASK_DESCRIPTION);
```
Comment (print-shanice) [inline src/main/java/seedu/address/logic/parser/AddTaskCommandParser.java:27]: Remember to edit the UG to reflect this error message

**Review by print-shanice — APPROVED** [review]
